src/gta/enums.hpp

```diff
@@ -485,7 +485,6 @@ enum class eNetworkEvents : uint16_t
 	REQUEST_DETACHMENT_EVENT,
 	KICK_VOTES_EVENT,
 	GIVE_PICKUP_REWARDS_EVENT,
-	NETWORK_CRC_HASH_CHECK_EVENT,
```

Comment:
How up-to-date is this enum as of now?
I a PR two month ago there was a discussion about it being outdated, several values changed. But idk what changed in the meantime: https://github.com/YimMenu/YimMenu/pull/2873#issuecomment-2022222753 and following comments.